bigtable/client/row_range.cc

```diff
@@ -52,6 +52,14 @@ bool RowRange::IsEmpty() const {
       return false;
   }
 
+  // Special case of an open interval of two consecutive strings.
+  // The only way for two strings to be consecutive is for the
+  // second to be equal to the first with an appended zero char.
+  if (start_open and end_open and (end->length() == start->length() + 1) and
+      (*(end->rbegin()) == '\0') and
+      (end->compare(0, start->length(), *start) == 0))
```

Comment:
Missing braces around the return.  Also, consider refactoring the more complicated parts of the boolean expression to an inline function (in the anonymous namespace), maybe something like:

```C++
inline bool Consecutive(std::string const& a, std::string const& b) { }
```

Reply:
Done.